src/odemis/driver/lakeshore.py

```diff
@@ -432,13 +430,14 @@ def __init__(self, timeout=1):
         self._input_buf = b""  # what we receive from the "host computer"
         self._status_byte = POWER_ON
 
-        self._setpoint = 10
-        self._temperature = 50
-        self._heating = 1
+        self._setpoint = 150
+        self._temperature = 100
+        self._stable_temperature = 50
```

Comment:
That's just a constant? Then make it a class constant and use upper case: `STABLE_TEMPERATURE`. Also add a small comment like "K, temperature reached without heating".